examples/flutter_gallery/lib/gallery/app.dart

```diff
@@ -74,18 +66,12 @@ class GalleryAppState extends State<GalleryApp> {
       },
     );
 
-    if (showPreviewBanner)
-      home = new Banner(
-        message: 'PREVIEW',
-        location: BannerLocation.topRight,
-        child: home,
-      );
-
-    if (config.updateUrlFetcher != null)
+    if (config.updateUrlFetcher != null) {
```

Comment:
style guide says braces around multi-line blocks, no braces around single-line blocks. sorry if i gave an incorrect comment in an earlier PR!